libpod/runtime.go

```diff
@@ -387,6 +387,12 @@ func SetXdgDirs() error {
 		return errors.Wrapf(err, "cannot set XDG_RUNTIME_DIR")
 	}
 
+	if rootless.IsRootless() && os.Getenv("DBUS_SESSION_BUS_ADDRESS") == "" {
+		if _, err := os.Stat(fmt.Sprintf("/run/user/%d/bus", rootless.GetRootlessUID())); err == nil {
+			os.Setenv("DBUS_SESSION_BUS_ADDRESS", fmt.Sprintf("unix:path=/run/user/%d/bus", rootless.GetRootlessUID()))
```

Comment:
Duplication bothers me; would you mind declaring a local var to hold `/run/user/etc` ?

Reply:
I've fixed it in the last commit.  I've also dropped the `/run/user/%d` part as we already have `runtimeDir`

Reply:
Let's fix this and then we can merge
```
buspath:=fmt.Sprintf("/run/user/%d/bus", rootless.GetRootlessUID()
if _, err := os.Stat(buspath); err := nil {
 			os.Setenv("DBUS_SESSION_BUS_ADDRESS",  fmt.Sprintf("unix:path=%s", buspath) 
}
```